README.md

```diff
@@ -45,10 +45,12 @@ Spyder is a Python development environment with a lot of features:
 
 * **Find in files**
 
-    Supporting regular expressions and mercurial repositories
+    Supporting regular expressions and Mercurial repositories
```

Comment:
Please remove `and Mercurial repositories`. That's not true anymore.

Reply:
Done